crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/reports/service/EmailNotificationServiceTest.java

```diff
@@ -74,7 +74,7 @@ void testSendEmailSuccessfully(int noOfInvokes, List<String> emailAddresses) thr
 
     }
 
-    private @NotNull JSONObject getExpectedJsonObject(String fileName) {
+    private @NonNull JSONObject getExpectedJsonObject(String fileName) {
```

Comment:
The @NonNull annotation is pretty pointless here since the method is private. I would just remove it.